Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -72,9 +73,33 @@ class SentrySessionReplay: NSObject {
         self.touchTracker = touchTracker
     }
     
+    // swiftlint:disable force_cast
+    public convenience init(replayOptions: SentryReplayOptions,
+                            replayFolderPath: URL,
+                            screenshotProvider: SentryViewScreenshotProvider,
+                            replayMaker: SentryReplayVideoMaker,
+                            breadcrumbConverter: SentryReplayBreadcrumbConverter,
+                            touchTracker: SentryTouchTracker?,
+                            dateProvider: SentryCurrentDateProvider,
+                            delegate: SentrySessionReplayDelegate,
+                            dispatchQueueGeneric: Any,
+                            displayLinkWrapperGeneric: Any) {
+        self.init(replayOptions: replayOptions,
+                  replayFolderPath: replayFolderPath,
+                  screenshotProvider: screenshotProvider,
+                  replayMaker: replayMaker,
+                  breadcrumbConverter: breadcrumbConverter,
+                  touchTracker: touchTracker,
+                  dateProvider: dateProvider,
+                  delegate: delegate,
+                  dispatchQueue: dispatchQueueGeneric as! SentryDispatchQueueWrapper,
+                  displayLinkWrapper: displayLinkWrapperGeneric as! SentryDisplayLinkWrapper)
```

Comment:
This convenience initialiser is more of a trick to avoid exposing the `SentryDispatchQueueWrapper` and `SentryDisplayLinkWrapper` types since this created more complications. I will revisit this for a better solution without a forced cast if needed.